app/helpers/application_helper.rb

```diff
@@ -4,4 +4,11 @@ def user_avatar(user, width: 32)
       image_tag(user.image, width: width)
     end
   end
+
+  def page_link_tree(category, discussion)
+    root_link = link_to("Conclave", categories_path)
+    category_link = link_to(category.name, category)
+    discussion_link = link_to(discussion.title, [category, discussion]) if discussion
+    [root_link, category_link, discussion_link].compact.join(" > ").html_safe
```

Comment:
Note: this will probably eventually need to be turned into a partial in order to create more complex HTML for better styling, but this will definitely work as a first-pass.